package.json

```diff
@@ -738,7 +739,7 @@
 		"source-map-support": "^0.4.0",
 		"through2": "^2.0.1",
 		"tslint": "^5.17.0",
-		"typescript": "^2.8.3",
+		"typescript": "^3.0.3",
```

Comment:
I suppose we can try to go further and bump Typescript to latest.

Reply:
I bumped Typescript dependency version up to v3.7.5.